### ISSUE TITLE
The VSCodeVim ext doesn't work when launching in a tmux session via CLI

### ISSUE THREAD

**Issue by niyue** [VSCodeVim/Vim#1314]
-----
### Update
This only happens when you launch vscode in a tmux session via CLI, caused by https://github.com/Microsoft/vscode/issues/13366

### What did you do?

I launch the vs code using CLI instead of clicking the app. For example:
```
$ code /path/to/some/folder
```

### What did you expect to happen?
I expect VSCodeVim to work as expected.

### What happened instead?
The vs code editor is launched, the cursor is a block, indicating it is in normal mode. However, I cannot move the cursor when pressing "hjkl", and pressing "i" doesn't make the editor to enter insert mode, it seems all of the keys are not functioning and don't provide any feedback, but the built-in vs code key shortcuts are still working, for example, when I press "cmd+p", the "Go to File" box will appear.

When launching the editor with GUI instead of CLI, the VSCodeVim works as expected.

### Technical details:

* VSCode Version: 1.9.1
* VsCodeVim Version: 0.5.3
* OS: OS X 10.12.3

**Comment by jpoon** [comment]
I do this all the time on MacOS and Windows, so I'm a bit skeptical about this actually being bug...

In any case, when you open Code through the CLI, is the vscodevim extension loaded? Are you passing any flags to code, especially `--disable-extensions`?

**Comment by niyue** [comment]
No, I don't pass any special flags to code, actually today is my first time use of Visual Studio Code when switching from (Sublime text + vim binding) to use (Visual Studio Code + VSCodeVim), and there may be something obvious I don't know but I simply downloaded the latest Visual Studio Code from the official website and installed the latest VSCodeVim extension and found this issue. 

I believe the VSCodeVim extension is loaded because I can see the "-- NORMAL MODE --" on the status bar at the bottom, which should be shown because of this extension.

**Comment by niyue** [comment]
@jpoon I think I found the cause. It is not caused by the bug of VSCodeVim, but caused by this one https://github.com/Microsoft/vscode/issues/13366 (it was closed but there are still updated in recent two weeks)

When launching vscode in a tmux session, you will run into the problem I mentioned above. Because I start tmux by default in my terminal, I got the false conclusion that VSCodeVim CLI integration is broken. In https://github.com/Microsoft/vscode/issues/13366, people are complaining about broken copy paste, which I also ran into. When I start the vscode out of tmux session, I don't see any issue. 

Feel free to close this issue. Thanks a lot.
